slack_bolt/util/utils.py

```diff
@@ -88,3 +88,9 @@ def get_name_for_callable(func: Callable) -> str:
 
 def get_arg_names_of_callable(func: Callable) -> List[str]:
     return inspect.getfullargspec(inspect.unwrap(func)).args
+
+
+def is_coroutine_function(func: Optional[Any]) -> bool:
```

Comment:
FYI I may rename `is_coroutine_function` to `is_callable_coroutine` in a follow up PR